### PR TITLE
Added numpy.order_statistics.quantile method

### LINKED ISSUE #19333: quantile

### PR DESCRIPTION
**Pull request by Ashwani132003**
Close #19333

### CODE REVIEW

**Comment by Ashwani132003** [comment]
casting to float64 will needed to be updated at frontends when ivy start supporting higher float precision dtypes .
image

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py
Comment (theRealBird) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py:1]: Any reason why we are taking only the first dimension of the axis in the test?

**Review by theRealBird — CHANGES_REQUESTED** [review]
Have left a comment for `quantile` at the moment. Thanks!

**Comment by theRealBird** [comment]
Hi @Ashwani132003,
Thank you for contributing to Ivy! Great work on the PRs! 
I also see that you have implemented `einsum` as a part of this PR? Was this intended?

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py
Comment (Ashwani132003) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py:1]: Hi, I referred it from torch_quantile. I suppose test function is designed to handle only a single-axis case. That's why added this.

**Comment by Ashwani132003** [comment]
> Hi @Ashwani132003, Thank you for contributing to Ivy! Great work on the PRs! I also see that you have implemented `einsum` as a part of this PR? Was this intended?

No, actually I am planning to do multiple contributions to this repo, initially I accidentally added commits from master/another branch. But now I learned to create commits from single branch. If there are any conflicts with it, then i can make a new pr if you want? Thanks

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py
Comment (Ashwani132003) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py:1]: image
this is from backend

**Comment by theRealBird** [comment]
> > Hi @Ashwani132003, Thank you for contributing to Ivy! Great work on the PRs! I also see that you have implemented `einsum` as a part of this PR? Was this intended?
> 
> No, actually I am planning to do multiple contributions to this repo, initially I accidentally added commits from master/another branch. But now I learned to create commits from single branch. If there are any conflicts with it, then i can make a new pr if you want? Thanks

Thanks a lot for investing your time at Ivy and doing multiple contributions. The only problem here is not the conflicts, but the problem of not linking the right issue in the PR description. The way you have linked #19333 should be followed for all the functions you are implementing. In this case, you would also have to create and then link and issue for einsum. I hope this clears it up.

Also, in the future, I would suggest making different PRs for different issues. This helps us quickly review and merge them and work on the implementations independently. Thanks a lot!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**